src/components/search.js

```diff
@@ -99,18 +92,19 @@ class Search extends Emitter {
                      * @event Chat#$"."error"."history
                      */
                     this.chatEngine.throwError(this, 'trigger', 'search', new Error('There was a problem searching history. Make sure your request parameters are valid and history is enabled for this PubNub key.'), status);
-
                 } else {
+                    const startDate = response.startTimeToken;
+                    this.referenceDate = response.startTimeToken;
+                    this.hasMore = response.messages.length === this.config.count && startDate !== '0';
 
-                    // timetoken of the first message in response
-                    this.firstTT = response.startTimeToken;
-                    // timetoken of the last message in response
-                    this.lastTT = response.endTimeToken;
+                    response.messages.sort((left, right) => (left.timetoken < right.timetoken ? -1 : 1));
 
-                    response.messages = this.sortHistory(response.messages);
+                    if (this.config.start && startDate < this.config.start) {
+                        this.hasMore = false;
+                        response.messages = response.messages.filter(event => event.timetoken >= this.config.start);
```

Comment:
I'd like to move the whole codebase to shorthand function calls in the future.

Reply:
What do you mean? Arrow functions? 
Should be something changed here?

Reply:
@ianjennings is there something I need to do here? Or it just some thought on future? To rewrite code to arrow functions usage?

Reply:
Haha, no just me thinking 💭